modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/PartitionReplicaLifecycleManager.java

```diff
@@ -190,6 +205,9 @@ public PartitionReplicaLifecycleManager(
         this.lowWatermark = lowWatermark;
         this.ioExecutor = ioExecutor;
         this.rebalanceScheduler = rebalanceScheduler;
+        this.clockService = clockService;
+        // should it be ExecutorInclinedPlacementDriver as in TableManager?
```

Comment:
Yes it should. And don't forget to remove the comment.

Reply:
Yes, it is. You may either extract ExecutorInclinedPlacementDriver from TableManager to some common place or just duplicate the class - no big deal in this particular case. Please fix it in follow up ticket.

Reply:
Made the [ticket](https://issues.apache.org/jira/browse/IGNITE-22928)